mantidproject/mantid
# Indirect - Elastic Window - Save Workspace Normalise by lowest temperature

**To test:**
Follow instructions 1 to 5 in pull request #20040, then click in `Save Result` and make sure that a Nexus `*_elt.nxs` is saved to the default save directory.

Fixes #20316

[Release Notes](https://github.com/mantidproject/mantid/blob/54fe0ed3e57518f3ff85e9a2404fa3df9e6c241a/docs/source/release/v3.11.0/indirect_inelastic.rst)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
